build.bat

```diff
@@ -94,12 +93,13 @@ COPY /Y /V "..\..\..\Changelog.txt"                      "temp_zip\"
 COPY /Y /V "Hunspellx86.dll"                             "temp_zip\"
 COPY /Y /V "Hunspellx64.dll"                             "temp_zip\"
 COPY /Y /V "SubtitleEdit.exe"                            "temp_zip\"
+COPY /Y /V "SubtitleEdit.exe.config"                     "temp_zip\"

```

Comment:
I could have sworn, the installer copies the config file, but, fact is, it does *not*.